distribution/dashboard/ui/src/pages/Cluster/Detail/Tenant/index.tsx

```diff
@@ -1,4 +1,60 @@
-// 某个集群下的租户列表
+import EventsTable from '@/components/EventsTable';
+import MonitorComp from '@/components/MonitorComp';
+import TenantsList from '@/pages/Tenant/TenantsList';
+import { getClusterDetailReq } from '@/services';
+import { getAllTenants } from '@/services/tenant';
+import { PageContainer } from '@ant-design/pro-components';
+import { useNavigate } from '@umijs/max';
+import { useRequest } from 'ahooks';
+import { Row } from 'antd';
+import BasicInfo from '../Overview/BasicInfo';
+import { getNSName } from '../Overview/helper';
+
+const defaultQueryRange = {
+  step: 20,
+  endTimestamp: Math.floor(new Date().valueOf() / 1000),
+  startTimestamp: Math.floor(new Date().valueOf() / 1000) - 60 * 30,
+};
+
 export default function Tenant() {
-  return <h1>Tenant</h1>;
+  const [ns, name, clusterName] = getNSName();
+  const navigate = useNavigate();
+  const { data: tenantsListResponse, run: getTenantsList } = useRequest(
+    getAllTenants,
+    {
+      manual: true,
+    },
+  );
+  const { data: clusterDetail } = useRequest(getClusterDetailReq, {
+    defaultParams: [{ name, ns }],
+    onSuccess: (res) => {
+      if (res?.info?.name) {
+        getTenantsList(res.name);
+      }
+    },
+  });
+  const tenantsList = tenantsListResponse?.data;
+  const handleAddCluster = () => navigate('/tenant/new');
+  return (
+    <PageContainer>
+      <Row gutter={[16, 16]}>
+        {clusterDetail && (
+          <BasicInfo {...(clusterDetail.info as API.ClusterInfo)} />
+        )}
+        {tenantsList && (
```

Comment:
This may not be a good design. If the cluster tenant is empty, you may need to display a module with no tenant data instead of not displaying it at all.